app/packs/src/apps/mydb/elements/details/samples/SampleDetails.js

```diff
@@ -739,59 +748,121 @@ export default class SampleDetails extends React.Component {
     )
   }
 
-  addManualCas(e) {
-    DetailActions.updateMoleculeCas(this.props.sample, e.value);
+  // addManualCas(e) {
+  //   DetailActions.updateMoleculeCas(this.props.sample, e.value);
+  // }
+
+  addHypenToCas(cas, boolean) {
```

Comment:
(Extremely) minor nitpick: multiple hyphens are added, hence the method could be called `addHyphens..` (plural) instead of `addHypen` (singular)? Sorry, I cannot help this :sweat_smile:

Reply:
you are right. fixed